### ISSUE TITLE
Bots and tag "donttakeitems" 

### ISSUE THREAD

**Issue by lockotar** [Regalis11/Barotrauma#9005]
I don't know if this is intended but when tagging diving suit lockers with "donttakeitems" bots can't get those suits. 
Probably every container tagged with this tag bots can't take.

As I understood, this tag should only prevent selling items from submarine containers and not prevent bots from taking it.

EDIT:
in issue #5431 "donttakeitems" also prevents bots from taking items. Should create a new tag for this problem.

Version
v0.17.12.0

**Comment by ekqvist** [comment]
The "donttakeitems" tag was originally introduced for AI, to prevent bots from taking items inside fabricators and constructors. When selling from sub was implemented for multiplayer, this tag was also additionally used to prevent selling items that could currently be used for crafting by other players.

However, the use of the tag isn't really self-explanatory at the moment. So, I added a new tag "dontsellitems" that can be used to prevent selling items inside containers that are tagged with it. Additionally, I added this tag to the fabricators and deconstructors so the vanilla behavior will remain the same. (Regalis11/Barotrauma-development@396a97f)

**Comment by Oukido** [comment]
Please remember to test old saves against the change. Similar to the issue with oxygensource tag on Oxygenite shards.

**Comment by Rokvach** [comment]
Is it possible to make this backwards compatible with older saves by making sure that items which has the donttakeitems also has the dontsellitems on older saves?

**Comment by ekqvist** [comment]
The vanilla items should automatically have the updated tags even if they're part of saves started in an earlier version. I think it's best that modders handle adding the tags to their own items themselves since there are also cases where you could want to have a container that the bots can't access but where you would be able to sell items from.

Let me know if there's any further issues with it or if I misunderstood something.

**Comment by Rokvach** [comment]
I am able to sell items from deconstrustors and fabricators on a older save, were both of the tags added to those items with this fix?

**Comment by ekqvist** [comment]
Fabricators and deconstructors already had the "donttakeitems" tag in the current Stable build (release branch). I started a new game on this version. I couldn't sell the rifle when it was inside a deconstructor. I saved the game and reloaded it on the dev build. Same thing, couldn't sell the rifle when it was inside a deconstructor. Switched to the sub editor and the deconstructor had the updated tags.

Either I'm misunderstanding the issue or I can't reproduce it.
